tools/rum-models-generator/Sources/CodeGeneration/Generate/Transformers/Swift/SwiftType.swift

```diff
@@ -0,0 +1,168 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-2020 Datadog, Inc.
+*/
+
+import Foundation
+
+/// Type-safe Swift schema.
+public protocol SwiftType {}
+
+/// Swift primitive type.
+public protocol SwiftPrimitiveType: SwiftType {}
+/// An allowed value of Swift primitive type.
+public protocol SwiftPrimitiveValue {}
+/// An allowed default value of Swift property.
+public protocol SwiftPropertyDefaultValue {}
+/// An allowed value of Swift with no obj-c interoperability.
+public protocol SwiftPrimitiveNoObjcInteropType: SwiftPrimitiveType {}
+
```

Comment:
The only changes in this file are `internal` → `public` access modifiers for all pieces of Swift schema. This is to make it visible for external module which performs code decoration.